compiler/luci/pass/src/QuantizedModelVerifier.test.cpp

```diff
@@ -236,6 +236,26 @@ class SoftmaxTestGraph final : public SimpleTestGraph
   luci::CircleSoftmax *_softmax = nullptr;
 };
 
+class SpaceToDepthTestGraph final : public SimpleTestGraph
+{
+public:
+  void init(void)
```

Comment:
as `SimpleTestGraph::init` is virtual, please add `override`.
I think @dayo09 will add omitted ones, it would be better to add for new ones.